os_installer2/pages/users.py

```diff
@@ -20,11 +20,13 @@
 DATA_COLUMN = 1
 
 
-class UserPanel(Gtk.VBox):
+class UserPanel(Gtk.Box):
     """Userpanel. Represents a user. Whoda thunk it. """
 
     def __init__(self, user):
-        Gtk.VBox.__init__(self)
+        Gtk.Box.__init__(self)
```

Comment:
Yea I was trying to be consistent with what I saw in the code. I'll make the change